src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -102,6 +102,9 @@ def cluster_projects(projects: Sequence[Project]) -> None:
                 tags={"clustered": False},
                 sample_rate=1.0,
             )
+            sentry_sdk.capture_message(
+                f"Transaction clusterer missed projects: {unclustered}", level="error"
```

Comment:
Can we keep the name  of the error always the same? 
```suggestion
                f"Transaction clusterer missed projects", level="error"
```

Which kind of makes it obvious what happens and use number of `unclustered` project as a additional data attached to the error?